src/android/GeofenceTransitionsIntentService.java

```diff
@@ -0,0 +1,224 @@
+/**
+2016 2 Symonhay.
+ */
+
+package com.example.scandevice;
+
+import android.app.IntentService;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.BitmapFactory;
+import android.graphics.Color;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.TaskStackBuilder;
+import android.text.TextUtils;
+import android.util.Log;
+
+import com.google.android.gms.location.Geofence;
+import com.google.android.gms.location.GeofencingEvent;
+
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Listener for geofence transition changes.
+ *
+ * Receives geofence transition events from Location Services in the form of an Intent containing
+ * the transition type and geofence id(s) that triggered the transition. Creates a notification
+ * as the output.
+ */
+public class GeofenceTransitionsIntentService extends IntentService{
+
+    protected static final String TAG = "GeofenceTransitionsIS";
+    private double lat, lon, alt;
+    private String timestamp = "yyyyMMdd'T'HH:mm:ss";
+
+    public DBHelper GPS_dbManager;
+    public Geofence geofence;
+    Location mLastLocation;
+    LocationManager locationManager = null;
+    public GetLocationInfo listener;
+
+    private static final int LOCATION_INTERVAL = 10000;
+    private static final float LOCATION_DISTANCE = 10f;
+
+
+
+    /**
+     * This constructor is required, and calls the super IntentService(String)
+     * constructor with the name for a worker thread.
+     */
+    public GeofenceTransitionsIntentService() {
+        // Use the TAG to name the worker thread.
+        super(TAG);
+    }
+
+    private class GetLocationInfo implements LocationListener{
+        @Override
+        public void onLocationChanged(Location location) {
+            lat = location.getLatitude();
+            lon = location.getLongitude();
+            alt = location.getAltitude();
+
+            SimpleDateFormat GPStime = new SimpleDateFormat("yyyyMMdd'T'HH:mm:ss");
+            timestamp = GPStime.format (location.getTime());
+        }
+
+        @Override
+        public void onStatusChanged(String s, int i, Bundle bundle) {}
+
+        @Override
+        public void onProviderEnabled(String s) {}
+
+        @Override
+        public void onProviderDisabled(String s) {}
+    }
+
+    private void initializeLocationManager() {
+        locationManager = (LocationManager) getSystemService(Context.LOCATION_SERVICE);
```

Comment:
as noted [earlier](https://github.com/miraculixx/cordova-rfscanner-plugin/pull/5#discussion_r53608603) this doesn't work.